### DISCUSSION TITLE
Suggestion Navigation Bar

### DISCUSSION THREAD

**Discussion by pixelFeliks** [z-huang/music#1293] [Ideas]
First of all, I Love Innertune. It is really amazing. 
As I am not using parts of the bottom navigation Bar It would be great If It was possible to take Out parts and sort them in a new Order. Possibly Like in the Screenshot.
![Screenshot_20240618-165606](https://github.com/z-huang/InnerTune/assets/173176969/6eeff152-f6d5-4f0f-aad0-c2cff121f439)
![Screenshot_20240618-165704](https://github.com/z-huang/InnerTune/assets/173176969/33725695-a1a3-4ba7-96ed-52952d651cb0)

**Comment by macharia22** [comment]
How do I unsubscribe from this thread

**Reply by pixelFeliks** [reply]
You could Look It up anywhere, really. But If you want to know. (in the App) click on the three dots in ther upper right corner and click unsubscribe.

**Comment by macharia22** [comment]
I have really tried to click unsubscribe both from here and github but I
keep getting these emails. I appreciate the support everyone is giving
innertune by pointing out bugs but I just want my emails to be minimal.
Thanks

**Comment by ketibanmeteor** [comment]
I think if the search button on navigation it would be good too
